app/controllers/UpscanController.scala

```diff
@@ -177,6 +183,7 @@ class UpscanController @Inject()(repository: UploadJourneyRepository,
                 logger.debug(s"[UpscanController][filePosted] - Success redirect called - file was previously in FAILED state, resetting state back to WAITING")
                 repository.updateStateOfFileUpload(journeyId, uploadModel).map(_ => NoContent.withHeaders(HeaderNames.ACCESS_CONTROL_ALLOW_ORIGIN -> "*"))
               } else {
+                PagerDutyHelper.log("filePosted",FILE_VERIFICATION_FAILURE_UPSCAN)
```

Comment:
I don't think this is an error case - this seems to be called when the upload hadn't previously failed - a valid case

Reply:
sure, removing the PagerDuty